### PR TITLE
Feature/general file updates

### PR DESCRIPTION
**Pull request by BooleanJulien**

### CODE REVIEW

**Review by zackt113 — APPROVED** [review]
Made some minor changes. Looks good overall

File: README.Rmd [hunk 1]
```diff
@@ -34,26 +43,30 @@ And the development version from [GitHub](https://github.com/) with:
...
-You'll still need to render `README.Rmd` regularly, to keep `README.md` up-to-date. `devtools::build_readme()` is handy for this. You could also use GitHub Actions to re-render `README.Rmd` every time you push. An example workflow can be found here: <https://github.com/r-lib/actions/tree/master/examples>.
+-   calculate_boot_stats: A confidence interval for a given sampling distribution is calculated for a given confidence level.
 
-You can also embed plots, for example:
+-   density_ci_plot: Generates a plot showing a density curve of a given sampling distribution along with its confidence interval ci for a given alpha.
 
-```{r pressure, echo = FALSE}
-plot(pressure)
-```
+-   ci_table: Generates a table that contains a given sampling distribution's mean and standard deviation along with relevant statistics like its confidence interval for a given confidence level.
```
Comment (voremargot) [inline README.Rmd]: Change the name of ci_table to summary_tables
```suggestion
-   summary_tables: Generates a table that contains a given sampling distribution's mean and standard deviation along with relevant statistics like its confidence interval for a given confidence level.
```

File: README.Rmd [hunk 2]
```diff
@@ -34,26 +43,30 @@ And the development version from [GitHub](https://github.com/) with:
...
-```{r cars}
-summary(cars)
-```
+-   bootstrap_distribution: A sampling distribution of \`rep\` replicates is generated for a specified estimator with replacement for a given bootstrap sample size.
 
-You'll still need to render `README.Rmd` regularly, to keep `README.md` up-to-date. `devtools::build_readme()` is handy for this. You could also use GitHub Actions to re-render `README.Rmd` every time you push. An example workflow can be found here: <https://github.com/r-lib/actions/tree/master/examples>.
+-   calculate_boot_stats: A confidence interval for a given sampling distribution is calculated for a given confidence level.
 
-You can also embed plots, for example:
+-   density_ci_plot: Generates a plot showing a density curve of a given sampling distribution along with its confidence interval ci for a given alpha.
```
Comment (voremargot) [inline README.Rmd]: Change the name to histogram_ci_plot
```suggestion
-   histogram_ci_plot: Generates a plot showing a density curve of a given sampling distribution along with its confidence interval ci for a given alpha.
```

File: DESCRIPTION [hunk 3]
```diff
@@ -1,10 +1,20 @@
...
-Version: 0.0.0.9000
-Authors@R: 
+Title: Bootstrapped Confidence Intervals, Plots, and More
+Version: 0.1.0
+Authors@R:
     person("First", "Last", , "first.last@example.com", role = c("aut", "cre"),
            comment = c(ORCID = "YOUR-ORCID-ID"))
-Description: What the package does (one paragraph).
+    person("Gordon", "Julien", "junigord@gmail.com", role = c("aut", "cre"))
+    person("Tang", "Zack", , "zacktang113@gmail.com", role = c("aut", "cre"))
```
Comment (voremargot) [inline DESCRIPTION]: add in me and gautham as authors in the description 
my email is vore.margot@gmail.com

**Review by voremargot — COMMENTED** [review]
Some small comments but otherwise looks good!

File: README.Rmd (hunk 2 above)
Comment (zackt113) [inline README.Rmd]:
```suggestion
-   histogram_ci_plot: Plots a bootstrapped sampling distribution with its 
    confidence interval and observed mean.
```

**Review by zackt113 — DISMISSED** [review]

File: DESCRIPTION (hunk 3 above)
Comment (BooleanJulien) [reply DESCRIPTION]: added

**Pull request merged**